src/Nordea/Components/AccordionMenu.elm

```diff
@@ -0,0 +1,63 @@
+module Nordea.Components.AccordionMenu exposing (OptionalConfig(..), Status(..), content, header, view)
+
+import Css exposing (alignItems, center, display, displayFlex, float, inlineBlock, justifyContent, lastChild, listStyle, none, nthLastChild, rem, right, spaceBetween, width)
+import Css.Global exposing (children, typeSelector, withAttribute)
+import Html.Styled as Html exposing (Attribute, Html, div)
+import Html.Styled.Attributes exposing (attribute, css)
+import Nordea.Resources.Icons as Icons
+
+
+type Status
+    = Open
+    | Closed
+
+
+type OptionalConfig
+    = Status Status
+
+
+view : List OptionalConfig -> List (Attribute msg) -> List (Html msg) -> Html msg
+view config attrs children_ =
+    let
+        config_ =
+            config
+                |> List.foldl
+                    (\attr acc ->
+                        case attr of
+                            Status x ->
+                                { acc | status = x }
+                    )
+                    { status = Open }
+
+        openAttr =
+            case config_.status of
+                Open ->
+                    attribute "open" "" :: attrs
+
+                Closed ->
+                    attrs
+
+        attrs_ =
+            css
+                [ children [ typeSelector "summary" [ listStyle none, children [ typeSelector "div" [ lastChild [ display inlineBlock ], nthLastChild "2" [ display none ] ] ] ] ]
+                , withAttribute "open" [ children [ typeSelector "summary" [ children [ typeSelector "div" [ lastChild [ display none ], nthLastChild "2" [ display inlineBlock ] ] ] ] ] ]
+                ]
+                :: openAttr
+    in
+    Html.details attrs_ children_
```

Comment:
It turns out the HTML tags you mentioned already handle opening and closing of the accordion. I eliminated the message from the component then. The effort was in customizing the chevron by changing visibility of elements in 2 states of `<details>` and `details open`.